src/app/catalogue/asset/components/Terms&ConditionsModal.jsx

```diff
@@ -0,0 +1,36 @@
+'use client'
+
+import { Button, Modal } from 'flowbite-react'
+import { useState } from 'react'
+
+export function TermsNConditions () {
```

Comment:
I don't know yet on Sedimark will deal with terms and conditions (and even how it will deal with payment), but I like your idea of the modal. Yet assuming these conditions needs to be accepted upon buying, I would suggest two possibilities:
- either we keep the modal, but it is triggered upon hitting the buying button, so we ditch the 'terms and conditions' hypertext.
- or we replace the modal by a checkbox with "I have read the Sedimark terms and conditions", keeping the hyperlink on "terms and conditions" (but we point so far to [sedimark website](https://sedimark.eu/)). That approach seems to be the most classical.

What do you think ?

Reply:
Im ok with both options, this modal as it is, comes from [Flowbite react](https://flowbite-react.com/docs/components/modal#:~:text=flowbite%2Dreact%22%3B-,Default%20modal,-%23), and when exploring the library I thought it could help us on this mvp :)

Reply:
Maybe let's open an issue to implement it when we have more information?

Reply:
Good idea. Issue created: #21